src/test/java/io/openshift/booster/HttpApplicationStandaloneIT.java

```diff
@@ -29,7 +29,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(VertxUnitRunner.class)
-public class HttpApplicationTest {
+public class HttpApplicationStandaloneIT {
```

Comment:
Why does this test become an IT?

That would avoid the failsafe configuration.